meta-resin-common/recipes-support/resin-device-api-key/resin-device-api-key/resin-device-api-key

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
I've merged them, now to see what the device team thinks of the yocto-ness of it :P